src/backend/index.js

```diff
@@ -1,6 +1,35 @@
-const http = require('http');
-
-http.createServer((_, res) => {
-    res.write('Blinkas web server');
-    res.end();
-}).listen(3000);
+/* eslint-disable */
+import Hapi from '@hapi/hapi';
+import pug from 'pug';
+import Vision from '@hapi/vision';
+import constants from './../../constants'
```

Comment:
Could we separate the absolute imports from the relative imports using a newline? This makes code cleaner and readable.